packages/ic-asset-certification/README.md

```diff
@@ -750,6 +750,16 @@ use ic_cdk::api::set_certified_data;
 set_certified_data(&asset_router.root_hash());
 ```
 
+It's also possible to delete all assets and their certification in one go:
+
+```rust
+# use ic_asset_certification::{Asset, AssetConfig, AssetFallbackConfig, AssetRouter, AssetRedirectKind, AssetEncoding};
```

Comment:
nit: why do we need to use all these objects when the code is using AssetRouter only?

Reply:
Copy pasta error, thanks!